resources/migrate-to-swift.md

```diff
@@ -53,13 +53,15 @@ Once the previous command is complete, run `pod install` or `pod update` to upda
 
 ## Update SDK initialization
 
-After you have imported the new Swift-based AEP-prefix libraries, you'll need to update SDK initialization code as described below. With Swift, the SDK has simplified initialization and registration of extensions to where the `MobileCore.start()` API is no longer required.
+After you have imported the new Swift-based AEP-prefix libraries, you'll need to update SDK initialization code as described below. With Swift,  the `MobileCore.start()` API is no longer required,  the SDK has simplified initialization and registration of extensions by calling `MobileCore.registerExtensions()` API with a completion block. After the given extensions have been registered, the SDK will be initialized and the completion block will be excuted.
 
```

Comment:
Should be two sentences.... "With Swift, the 'MobileCore.start()' API is no longer required. The SDK..."

Also, looks like an extra space got added after "With Swift".

Looks good otherwise.

Reply:
I have one small suggestion to make the sentence a bit more grammatically correct. And then i added something at the end to help clarify that code which used to be in start block now gets moved to the new completion block. This is what the customer was confused about. 

The SDK has simplified initialization and registration of extensions by providing the `MobileCore.registerExtensions()` API. After the given extensions have been registered, the SDK will be initialized and the completion block will be executed. Code which used to reside in the `start()` block will now reside in the `MobileCore.registerExtensions()` completion block.